nautobot/core/management/commands/build_ui.py

```diff
@@ -0,0 +1,141 @@
+            jsconfig = json.load(base_config_file)
+
+        # We're going to modify this list if apps don't have a `ui` directory.
+        enabled_apps = copy.copy(settings.PLUGINS)
+
+        for app_class_path in settings.PLUGINS:
+            app_name = app_class_path.split(".")[-1]
+            app_config = apps.get_app_config(app_name)
+            abs_app_path = Path(app_config.path).resolve()
+            abs_app_ui_path = abs_app_path / "ui"
+            app_path = Path(os.path.relpath(abs_app_path, ui_dir))
+            app_ui_path = app_path / "ui"
+
+            # Assert that an App has a UI folder.
+            if not abs_app_ui_path.exists():
+                self.stdout.write(self.style.ERROR(f"- App {app_name!r} does not publish a UI; Skipping..."))
+                enabled_apps.remove(app_class_path)
+                continue
+            self.stdout.write(self.style.SUCCESS(f"- App {app_name!r} imported"))
+
+            jsconfig["compilerOptions"]["paths"][f"@{app_name}/*"] = [f"{app_ui_path}/*"]
+
+        with open(jsconfig_file_path, "w", encoding="utf-8") as generated_config_file:
+            json.dump(jsconfig, generated_config_file, indent=4)
+
+        app_imports_final_file_path = Path(ui_dir, "src", "app_imports.js")
+        environment = jinja2.sandbox.SandboxedEnvironment(loader=jinja2.FileSystemLoader(ui_dir))
+        template = environment.get_template("app_imports.js.j2")
+        content = template.render(apps=enabled_apps)
+
+        with open(app_imports_final_file_path, "w", encoding="utf-8") as generated_import_file:
+            generated_import_file.write(content)
+
+        # Touch the router to attempt to trigger a server reload.
+        Path(router_file_path).touch()
+
+    def run_command(self, command, message, cwd=settings.NAUTOBOT_UI_DIR):
+        """
+        Run a `command`, displaying `message` and exit. This splits it for you and runs it.
+
+        Args:
+            command (str): The command to execute
+            message (str): Message to display when command is executed
+        """
+        self.stdout.write(self.style.WARNING(message))
+
+        try:
+            out = subprocess.run(
+                shlex.split(command),
+                check=False,
+                cwd=cwd,
+                env={**os.environ.copy()},
+            )
+        except FileNotFoundError as err:
+            raise CommandError(f"`{command}` failed with error: {err}")
+
+        if out.returncode:
+            raise CommandError(f"`{command}` failed with exit code {out.returncode}")
+
+    def handle(self, *args, **options):
+        verbosity = options["verbosity"]
+
+        # NAUTOBOT_DEBUG is also set here so that `console.log` isn't suppressed when we want
+        # verbosity.
+        if verbosity > 1:
+            os.environ["NAUTOBOT_DEBUG"] = "1"
+
+        verbosity_map = {
+            0: "silent",
+            1: "silent",  # Default is "notice" but still too chatty, so "silent".
```

Comment:
What about `warn` or `error`? Still too chatty?

Reply:
Warn/error are less than notice and yea, still too chatty for the default. I can demonstrate.